CHANGELOG.md

```diff
@@ -0,0 +1,317 @@
+# Change Log
+
+All notable changes to "Azure Toolkit for IntelliJ IDEA" will be documented in this file.
+
+- [Change Log](#change-log)
+
+  - [[3.19.0]](#3190)
```

Comment:
How about to remove `[ ]` around the version? it looks weird.

Reply:
Done~